base/.eslintrc.json

```diff
@@ -56,6 +57,44 @@
     "default-case": 0,
     "no-plusplus": 0,
     "dot-notation": 0,
-    "eol-last": 2
+    "eol-last": 2,
+
+    "import/no-unresolved": [
+      2,
+      {
+        "commonjs": true,
+        "amd": true
+      }
+    ],
+    "import/no-self-import": 2,
+    "import/no-cycle": 2,
+    "import/no-useless-path-segments": ["error", {
+      "noUselessIndex": true
+    }],
+    "import/no-unused-modules": [2, {
+      "unusedExports": true
+    }],
+    "import/no-named-as-default": 2,
```

Comment:
Not a fan of this rule. I personally don't agree with it, and I don't think we'll ever be able to enable it on any projects due to existing naming conventions.

Reply:
Removed